pkg/kubevirt/machine_test.go

```diff
@@ -212,3 +237,14 @@ func (e FakeVMCommandExecutor) ExecuteCommand(command string) (string, error) {
 		return "", errors.New("unexpected input argument")
 	}
 }
+
+func FakeNewMachine(ctx *context.MachineContext, client client.Client, vmExecutor FakeVMCommandExecutor, sshPubKey []byte) (*Machine, error) {
```

Comment:
Consider change this to mock auto generated by GoMock, can be done after this PR merge

Reply:
I think this was mostly bad function naming on my part. I changed this up a bit. We need a "real" machine object here, we just need it setup for the test driver in a specific way that allows certain parts to get mocked. I changed this function to `defaultTestMachine`